tsfeatures/tests/test_sparsity.py

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Please add these tests to nbs/tsfeatures.ipynb

Reply:
added the tests to the tsfeatures.ipynb